opentelemetry-resourcedetector-gcp/src/opentelemetry/resourcedetector/gcp_resource_detector/__init__.py

```diff
@@ -99,12 +99,74 @@ def get_gke_resources():
     return common_attributes
 
 
+def get_cloudrun_resources():
+    """Resource finder for Cloud Run attributes"""
+
+    if os.getenv("K_CONFIGURATION") is None:
+        return {}
+
+    (
+        common_attributes,
+        all_metadata,
+    ) = _get_google_metadata_and_common_attributes()
+
+    faas_name = os.getenv("K_SERVICE")
+    if faas_name is not None:
+        common_attributes["faas.name"] = str(faas_name)
+
+    faas_version = os.getenv("K_REVISION")
+    if faas_version is not None:
+        common_attributes["faas.version"] = str(faas_version)
+
+    common_attributes.update(
+        {
+            "cloud.platform": "gcp_cloud_run",
+            "cloud.region": all_metadata["instance"]["region"].split("/")[-1],
+            "faas.id": all_metadata["instance"]["id"],
+            "gcp.resource_type": "cloud_run",
```

Comment:
I'm not familiar with `gcp.resource_type`.  It seems like duplicate information with cloud.platform.  Why is it included?

Reply:
It predates `cloud.platform` so it was our hack to make things work. We should get rid of it, see #85